src/status_im/navigation/core.cljs

```diff
@@ -37,14 +37,14 @@
 
   (navigation/reg-component-did-appear-listener
    (fn [view-id]
-     (when (get views/screens view-id)
-       ;;NOTE when back from the background on Android, this event happens for all screens, but we
```

Comment:
now we know what this comment meant :D 
https://github.com/status-im/status-mobile/issues/19276 @flexsurfer @briansztamfater